.github/workflows/release.yaml

```diff
@@ -24,3 +24,20 @@ jobs:
           ORG_GRADLE_PROJECT_mavenCentralPassword: ${{ secrets.MAVEN_CENTRAL_PASSWORD }}
           ORG_GRADLE_PROJECT_signingInMemoryKey: ${{ secrets.SIGNING_KEY }}
         run: ./gradlew build publish
+
+  publish-mips:
+    needs: publish
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 19
+      - name: Build and publish mips artifacts
+        env:
+          ORG_GRADLE_PROJECT_mavenCentralUsername: ${{ secrets.MAVEN_CENTRAL_USERNAME }}
+          ORG_GRADLE_PROJECT_mavenCentralPassword: ${{ secrets.MAVEN_CENTRAL_PASSWORD }}
+          ORG_GRADLE_PROJECT_signingInMemoryKey: ${{ secrets.SIGNING_KEY }}
+        run: ./gradlew build publishMips -x check
```

Comment:
I'm not sure I'd bother with running `build` (and needing to exclude `check` here).

I think instead we make this job depend on the other one so that we know `build` has already succeeded and we are only concerned with publishing these two targets.

Reply:
If you want to split the jobs, we need to cache the build artifact from the other job (we are not doing step both run in 2 independent containers), which is more effort. (I did that [here](https://github.com/bitPogo/kfixture/blob/main/.github/workflows/ci-release.yml) and you will need [this](https://github.com/bitPogo/workflows/tree/main/.github/workflows) as well to understand it).

The check we can skip since a) it did run already with MacOS and b) the container will not execute the tests since it does not run on MIPS.

Reply:
Ah, looks like you already have the `needs: publish`. I must have missed it first time around because I was reviewing on mobile. My bad.

So yeah the only thing left is why do `build publishMips -x check` instead of only `publishMips`?

Reply:
My bad...you are absolutely right.